docs/environment-setup.md

```diff
@@ -168,6 +168,14 @@ _Note: If experiencing an error such as `Error starting userland proxy: listen t
 
 To run Elasticsearch natively, follow the instructions for your OS [here](https://www.elastic.co/guide/en/elastic-stack-get-started/7.6/get-started-elastic-stack.html#install-elasticsearch)
 
-### Finally
+### Finally:
 
-Run `npm start` and open `localhost:3000`
+Telescope requires a running back-end and front-end to start. The following steps assume you are running the back-end locally. For more information on running the front-end with our staging/production as the back-end, please refer to our [Front-End Document](gatsbyjs.md).
+
+Start the back-end for Telescope:
+Run `npm start`
+
+Build the front-end for Telescope:
+Run `npm run build`
+
+Open `localhost:3000`
```

Comment:
I think you have to setup the API proxy for this, no?  Shouldn't we send them to `localhost:8000`?  Correct me if I'm out of date on this.

Reply:
I think `8000` is only for gatsby's development server (`npm run develop`). With `npm run build`, we make a production build that will be served by telescope (that's why port `3000` is used in this case).